src/util/accessibility/live.ts

```diff
@@ -33,6 +33,10 @@ function getLiveElement(document: any, lazyCreate = false): HTMLElement | null {
 
 
 
+// TODO: Add Angular decorator.
+// TODO: Add Angular decorator.
+// TODO: Add Angular decorator.
+// TODO: Add Angular decorator.
```

Comment:
This is due to https://github.com/angular/angular/issues/37726 I believe so in this sense this is Angular migrations "fault". Still, we should remove those TODOs from this PR.

Reply:
Ooops, missed those